### PR TITLE
Add ~arm64 support to dev-lang/rust and dev-util/cargo

### PR DESCRIPTION
**Pull request by psychedup**
I was able to build both of these with a Pine64 (2GB model), and then successfully merged =www-client/firefox-58.0.1 on the same thing.

This is my first pull request ever. Let me know if I missed anything.

### CODE REVIEW

File: dev-lang/rust/rust-1.23.0-r1.ebuild [hunk 1]
```diff
@@ -19,15 +19,17 @@ else
 	SLOT="stable/${ABI_VER}"
 	MY_P="rustc-${PV}"
 	SRC="${MY_P}-src.tar.gz"
-	KEYWORDS="~amd64 ~x86"
+	KEYWORDS="~amd64 ~x86 ~arm64"
```
Comment (dracwyrm) [inline dev-lang/rust/rust-1.23.0-r1.ebuild]: In alphabetical order please. Also, are you sure this is not compatible with 32 bit arm?

File: dev-util/cargo/cargo-0.24.0.ebuild [hunk 2]
```diff
@@ -143,19 +143,24 @@ SRC_URI="https://github.com/rust-lang/cargo/archive/${PV}.tar.gz -> ${P}.tar.gz
...
+	)
+	arm64? (
+		https://static.rust-lang.org/dist/cargo-${BOOTSTRAP_VERSION}-aarch64-unknown-linux-gnu.tar.gz
 	)"
 
 RESTRICT="mirror"
 LICENSE="|| ( MIT Apache-2.0 )"
 SLOT="0"
-KEYWORDS="~amd64 ~x86"
+KEYWORDS="~amd64 ~x86 ~arm64"
```
Comment (dracwyrm) [inline dev-util/cargo/cargo-0.24.0.ebuild]: ditto

**Comment by psychedup** [comment]
Ok, I put the keywords in order.

I do not have a 32-bit arm machine to test on.

**Comment by dracwyrm** [comment]
I have 64 bit arm as well. I need to see if I can dual boot it sometime so I have my Gentoo test environment.

**Pull request closed without merging**